tasks/kmt.py

```diff
@@ -1372,7 +1372,7 @@ def clean(ctx: Context, stack: str | None = None, container=False, image=False):
         stack
     ), f"Stack {stack} does not exist. Please create with 'inv kmt.create-stack --stack=<name>'"
 
-    ctx.run("rm -rf ./test/kitchen/site-cookbooks/dd-system-probe-check/files/default/tests/pkg")
+    ctx.run("rm -rf ./test/files/default/tests/pkg")
```

Comment:
This used to be `{KITCHEN_ARTIFACT_DIR}/pkg`, should it now be `{E2E_ARTIFACT_DIR}/pkg`?